### PR TITLE
Optimize loops and template strings

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: lib/files.js [hunk 1]
```diff
@@ -187,21 +187,21 @@ impress.dirIntrospect = (application, relPath, callback) => {
           const ext = api.common.fileExt(name);
           let method = HANDLER_TYPES[ext]; // 'unknown'
           if (ext === 'json') {
-            impress.HTTP_VERBS.forEach(verb => {
+            for (const verb in impress.HTTP_VERBS) {
```
Comment (nechaido) [inline lib/files.js]:
```suggestion
            for (const verb of impress.HTTP_VERBS) {
```

File: lib/firewall.js [hunk 2]
```diff
@@ -62,18 +62,17 @@ firewall.check = client => {
   let allowed = true;
   let limited = false;
 
-  let key, limit;
-  firewall.objects.map(obj => {
+  for (const obj in firewall.objects) {
```
Comment (nechaido) [inline lib/firewall.js]:
```suggestion
  for (const obj of firewall.objects) {
```

File: lib/core.js [hunk 3]
```diff
@@ -435,7 +435,7 @@ const startServers = () => {
 
   let workerId = 0;
 
-  serverNames.forEach(serverName => {
+  for (const serverName in configServers) {
```
Comment (nechaido) [inline lib/core.js]:
```suggestion
  for (const serverName of configServers) {
```

File: lib/cache.js [hunk 4]
```diff
@@ -137,6 +137,7 @@ class Cache {
     this.application.emit('change');
     const updates = this.updates;
     this.updates = [];
+    console.dir(updates);
```
Comment (nechaido) [inline lib/cache.js]: Is it a debug log?

File: lib/core.js (hunk 3 above)
Comment (tshemsedinov) [reply lib/core.js]: in this case: `for (const serverName of serverNames) {`

File: lib/cache.js (hunk 4 above)
Comment (tshemsedinov) [reply lib/cache.js]: yes :smile:

**Comment by tshemsedinov** [comment]
Landed in 3c9fc6f

**Pull request closed without merging**